permissions/plugin-ansicolor.yml

```diff
@@ -10,3 +10,4 @@ developers:
 - "qvicksilver"
 - "joemerten"
 - "mkobit"
+- "dnusbaum"
```

Comment:
LDAP intentionally distinct from GH user name, right?

Reply:
Yep, I would like to change my GitHub username to match but have not had time to reach out to GH support to see if they would give me https://github.com/dnusbaum, and I would need to double-check the specifics of what gets redirected and what doesn't.